torch_uncertainty/routines/classification.py

```diff
@@ -205,8 +217,14 @@ def criterion(self) -> nn.Module:
             self.loss = partial(self.loss, model=self.model)
         return self.loss()
 
-    def forward(self, inputs: Tensor) -> Tensor:
-        return self.model.forward(inputs)
+    def forward(self, inputs: Tensor, return_features: bool = False) -> Tensor:
```

Comment:
What do you think about this part @alafage?

Reply:
I think it's a clean and efficient way to do it! Don't forget to update the signature accordingly:
```py
-> tuple[Tensor, Tensor | None]:
```

Reply:
Yes, thx!